src/metric/elementwise_metric.cu

```diff
@@ -197,6 +197,20 @@ struct EvalRowLogLoss {
   }
 };
 
+struct EvalRowMyLoss {
```

Comment:
What is this for?

Reply:
This should not be there. It was part of a practice exercise. Will remove.